### ISSUE TITLE
Under Python 3, get_n_primes_near_x returns a float.

### ISSUE THREAD

**Issue by ctb** [dib-lab/khmer#1349]
This should be changed to an integer, as in Python 2.

**Comment by luizirber** [comment]
Do you have a test case? Seems to work here:

Python 3.5.1 (default, Jan 11 2016, 11:34:42)
[GCC 4.2.1 Compatible Apple LLVM 7.0.2 (clang-700.1.81)] on darwin

``` python
>>> p = khmer.get_n_primes_near_x(5, 100000000000000000)
>>> print(p)
[99999999999999997, 99999999999999977, 99999999999999961, 99999999999999943, 99999999999999919]
>>> type(p[0])
int
```

**Comment by luizirber** [comment]
Oh, I see: if you give target as a float, it returns a float instead ([example][1]).

See https://github.com/dib-lab/khmer/compare/fix/1349 for a test case and possible solution.

[1]: https://github.com/ctb/sourmash/blob/a70e61808c7b3e739d152b1434c9666976fe607e/sourmash.py#L35

**Comment by ctb** [comment]
Being fixed in #1350.

**Comment by betatim** [comment]
Closed by #1350
